rules/windows/persistence_startup_folder_scripts.toml

```diff
@@ -116,7 +116,10 @@ timestamp_override = "event.ingested"
 type = "eql"
 
 query = '''
-file where host.os.type == "windows" and event.type != "deletion" and user.domain != "NT AUTHORITY" and
+file where host.os.type == "windows" and event.type != "deletion" and
+
+  file.extension : ("lnk", "vbs", "vbe", "wsh", "wsf", "js") and
+  not user.domain : ("NT *", "* NT") and
```

Comment:
We had comments about wildcards in the beginning of strings and performance concerns.
Should we do `not (startsWith(user.domain, "NT") or endsWith(user.domain, "NT"))`

Reply:
Added